### PR TITLE
[prebuild-logs] ignore other instances

### PR DESCRIPTION
**Pull request by AlexTugarev**
## Description
Prebuild logs should not react on instance updates of other workspaces. 

## Related Issue(s)

Ref #5888

## How to test

See https://github.com/gitpod-io/gitpod/issues/5888#issuecomment-936076775

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by jankeromnes — APPROVED** [review]
Excellent catch 👏 and many thanks for the quick fix 🙏 

Code change looks good and seems to work as intended. ✅

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: fd428e3202fa778d6aef22dd33602f9608e23656</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6066#pullrequestreview-772584790" title="Approved">jankeromnes</a>*

Associated issue: *#5888*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
